src/components/networked.js

```diff
@@ -267,6 +273,18 @@ AFRAME.registerComponent('networked', {
     if (entityData[0] == 1) {
       entityData = Compressor.decompressSyncData(entityData, this.getAllSyncedComponents());
     }
+
+    // Avoid updating components if the entity data received did not come from the current owner.
+    if (entityData.lastOwnerTime < this.lastOwnerTime || 
+          (this.lastOwnerTime === entityData.lastOwnerTime && this.data.owner > entityData.owner)) {
```

Comment:
I'm not well versed with this code, but aren't `data.owner` and `entityData.owner` client ids? If so why are they being compared with a `>` operator here and not `!=`?

Reply:
I belive this is to handle the unlikely case where two people take ownership at the exact same timestamp. In this case the client with the larger id wins. This is just to ensure that everything remains eventually consistent.

Reply:
@gfodor you're right that `data.owner` will be a clientId. The type of clientId is set by the adapter ([passed into the `connectSuccess` callback](https://github.com/netpro2k/networked-aframe/blob/a9107247f9e82145d9fce75fe4ab764791fb02dc/src/NetworkConnection.js#L66) so as long as that type supports the `>` operator this will work as expected. Restricting clientId's to strings and numbers seems reasonable.

Reply:
Actually regardless of all that the Networked component schema restricts the owner to a string.